docs/calendar.md

```diff
@@ -0,0 +1,277 @@
+
+> **NOTE:** The Temporal polyfill currently only includes the ISO 8601 calendar, but the Temporal proposal will eventually provide all the calendars supported by Intl.
+> See [issue #541](https://github.com/tc39/proposal-temporal/issues/541).
+
+### When to use the `Temporal.Calendar`
+
+Most code will not have to use the `Temporal.Calendar` API.
+In general, it's enough to use the ISO 8601 calendar in your code, and output dates in the end user's preferred calendar using `Temporal.DateTime.toLocaleString()`, `Temporal.Date.toLocaleString()`, etc.
+
+However, the `Temporal.DateTime`, `Temporal.Date`, `Temporal.YearMonth`, and `Temporal.MonthDay` objects can all be expressed in a different calendar.
+If you need to do calculations in another calendar, then you will need to make your Temporal objects use that calendar.
+
+For example, to add a month to a date in the Hebrew calendar:
+```javascript
+date.withCalendar('hebrew').plus({ months: 1 })
+```
+
+## Constructor
+
+### **new Temporal.Calendar**(_calendarIdentifier_: string) : Temporal.Calendar
+
+**Parameters:**
+- `calendarIdentifier` (string): An identifier for the calendar.
+
+**Returns:** a new `Temporal.Calendar` object.
+
+For a list of calendar identifiers, see the documentation for [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#Parameters).
+If `calendarIdentifier` is not a known calendar, then a `RangeError` is thrown.
+
+Use this constructor directly if you have a string that is known to be a correct calendar identifier.
+If you have an ISO 8601 date-time string with a `[c=identifier]` annotation, then `Temporal.Calendar.from()` is probably more convenient than parsing the identifier out of the string.
+
+Example usage:
+```javascript
+cal = new Temporal.Calendar('iso8601');
+cal = new Temporal.Calendar('gregory');
+/*⚠️*/ cal = new Temporal.Calendar('discordian');  // not a built-in calendar, throws
+```
+
+## Static methods
+
+### Temporal.Calendar.**from**(_thing_: any) : Temporal.Calendar
+
+**Parameters:**
+- `thing`: A `Temporal.Calendar` object or a value from which to create a `Temporal.Calendar`.
+
+**Returns:** a new `Temporal.Calendar` object.
+
+This static method creates a new calendar from another value.
+If the value is another `Temporal.Calendar` object, a new object representing the same time zone is returned.
+
+Any other value is converted to a string, which is expected to be either:
+- a string that is accepted by `new Temporal.Calendar()`; or
+- a string in the ISO 8601 format.
+
+Note that the ISO 8601 string can be extended with a `[c=identifier]` annotation in square brackets appended to it.
+Without such an annotation, the calendar is taken to be `iso8601`.
+
+This function is often more convenient to use than `new Temporal.Calendar()` because it handles a wider range of input.
```

Comment:
It's not just more convenient; it's what users should prefer, since custom calendars are only injected into the `.from` method, not the constructor.

Reply:
Fixed in #636, will rebase.